packages/client-ts/src/index.ts

```diff
@@ -2,6 +2,12 @@ import Client from './client'
 import Network from './network'
 import Context from './context/context'
 import Messaging from './context/messaging'
+import Datastore from './context/datastore'
+import Database from './context/database'
+import Notification from './context/notification'
+import { Profile } from './context/profiles/profile'
+import * as DbRegistry from './context/db-registry'
```

Comment:
I can understand Notification, Profile and the DbRegistry classes. I wasn't sure about them anyway. 

But this whole issue arose because of `Database`. I was forced to do this to get typechecking:

```
import Database from "@verida/client-ts/dist/context/database";
```

Reply:
I've pushed an update that removes Notification, Profile and the DbRegistry from the export but keeps Database. 

I won't mark this comment as resolved though.

Reply:
Added `Profile` back after discussion on Discord:

> tahpot | Verida — Profile and Database make sense to me as they will possibly be used by apps.

Marking this as resolved.